app/src/main/java/com/example/internetshop/presentation/activity/MainActivity.kt

```diff
@@ -109,14 +101,23 @@ class MainActivity : AppCompatActivity(), ContainerHolder {
                 R.id.cartFragment
             )
         ).build()
+        binding?.let {
+            it.bottomNavBar.setupWithNavController(navController!!)
+            it.toolbar.setNavigationIcon(R.drawable.ic_back_arrow)
+            it.collapsingLayout.setupWithNavController(
+                it.toolbar,
+                navController!!,
```

Comment:
To avoid non safe calls like `navController!!` and `appBarConfig!!` pass there objects as non-null params in this method.  Also, if you pass them as params, then you don't need global nullable fields. Try to do not use non safe calls `!!`

Reply:
fixed